data/src/main/java/com/linkedin/data/codec/ProtobufDataCodec.java

```diff
@@ -62,10 +62,13 @@
   private static final byte RAW_BYTES_ORDINAL = 10;
   private static final byte NULL_ORDINAL = 11;
   private static final byte ASCII_STRING_LITERAL_ORDINAL = 20;
+  private static final byte FIXED_FLOAT_ORDINAL = 21;
+  private static final byte FIXED_DOUBLE_ORDINAL = 22;
 
   protected final SymbolTable _symbolTable;
 
   protected final boolean _supportsASCIIOnlyStrings;
+  protected final boolean _useFixedLengthEncodingForFloatingPointValues;
```

Comment:
can you rename this to something shorter? like useFixedFloatDouble or useFixedDecimalValue etc

Reply:
Done